nox/manifest.py

```diff
@@ -319,7 +319,7 @@ def notify(
         # Locate the session in the list of all sessions, and place it at
         # the end of the queue.
         for s in self._all_sessions:
-            if s == session or s.name == session or session in s.signatures:
+            if s == session or s.name == session or session in s.signatures:  # noqa: PLR1714
```

Comment:
Probably should report this one to Ruff. It's not actually correct, since we are relying on short-circuiting here.

Reply:
Actually, maybe it's fine. I think s.name has to be valid, and session is either a string or a session.